getting-started/proc-migrating-oauth2-proxy-to-oidc.adoc

```diff
@@ -0,0 +1,76 @@
+[id="proc-migrating-oauth2-proxy-to-oidc_{context}"]
+
+= Migrating from OAuth2 Proxy with Keycloak to OIDC in {product-short}
+If you are using OAuth2 Proxy as an authentication provider with Keycloak, and you want to migrate to OIDC, you can update your authentication provider configuration to use OIDC.
+
+.Procedure
+
+. In Keycloak, update the valid redirect URI to `https://<rhdh_url>/api/auth/oidc/handler/frame`. Ensure to replace `<rhdh_url>` with your {product-short} application URL, such as, `my.rhdh.example.com`.
+. Replace the `oauth2Proxy` configuration values in the `auth` section of your `app-config.yaml` file with the `oidc` configuration values.
++
+An example of the `auth` configuration for `oauth2Proxy` is as follows: 
++
+[source,yaml]
+----
+auth:
+  environment: production
+  session:
+    secret: ${SESSION_SECRET}
+  providers:
+    oauth2Proxy: {}
+      production:
+        metadataUrl: ${KEYCLOAK_METADATA_URL}
+        clientId: ${KEYCLOAK_CLIENT_ID}
+        clientSecret: ${KEYCLOAK_CLIENT_SECRET}
+        prompt: ${KEYCLOAK_PROMPT} # recommended to use auto
+
```

Comment:
This just needs to be removed as oauth2Proxy doesn't support and values underneath it.